lib/hypertypes/README.md

```diff
@@ -3,9 +3,11 @@
 Generate the requisite Python files by running:
 
 ```bash
-sh scripts/build_type_files.sh path/to/ABIs num_chars_per_line
+sh pypechain path/to/ABIs --output_dir path/to/temp_dir
 ```
 
+and then moving the generated files to lib/hypertypes/hypertypes
```

Comment:
should it be "and then move ..."?

Reply:
The complete sentence sounds right to me:

Generate the requisite Python files by running:

```bash
sh pypechain path/to/ABIs --output_dir path/to/temp_dir
```

and then moving the generated files to lib/hypertypes/hypertypes